server/libs/infra/src/repositories/media.repository.ts

```diff
@@ -31,6 +37,35 @@ export class MediaRepository implements IMediaRepository {
     }
   }
 
+  async resizeRaw(input: string, output: string, options: ResizeRawOptions): Promise<void> {
+    try {
```

Comment:
Linking https://github.com/immich-app/immich/pull/2198#discussion_r1164776892 again as it's marked as out-dated.